libs/payments/stripe/src/lib/factories/invoice-preview.factory.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from '@faker-js/faker';
+import { InvoicePreview } from '../stripe.types';
+
+export const StripeInvoicePreviewFactory = (
```

Comment:
Could you rename this to `InvoicePreviewFactory` instead? This way it's more clear that this is a factory for a custom type that's not related to the Stripe SDK or Stripe Client wrappers.

Alternatively, having a separate file for all custom type factories could also make that separation more clear. I was thinking `lib/stripe.factories.ts`.